app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,26 @@
+<footer class="footer" role="contentinfo">
+  <div class="footer-logo">
+    <%= image_tag "logo_symbol.png"%>
+  </div>
+  <div class="footer-links">
+    <ul>
+      <li><h3>Navigate</h3></li>
+      <li><%= link_to t("shared.header.books"), books_path %></li>
```

Comment:
I would say that since the header isn't actually an item in the list, it should be outside of the `<ul>`?